src/ansi-c/parser.y

```diff
@@ -50,223 +56,223 @@ extern char *yyansi_ctext;
 
 /*** ANSI-C keywords ***/
 
-%token TOK_AUTO      "auto"
-%token TOK_BOOL      "bool"
-%token TOK_COMPLEX   "complex"
-%token TOK_BREAK     "break"
-%token TOK_CASE      "case"
-%token TOK_CHAR      "char"
-%token TOK_CONST     "const"
-%token TOK_CONTINUE  "continue"
-%token TOK_DEFAULT   "default"
-%token TOK_DO        "do"
-%token TOK_DOUBLE    "double"
-%token TOK_ELSE      "else"
-%token TOK_ENUM      "enum"
-%token TOK_EXTERN    "extern"
-%token TOK_FLOAT     "float"
-%token TOK_FOR       "for"
-%token TOK_GOTO      "goto"
-%token TOK_IF        "if"
-%token TOK_INLINE    "inline"
-%token TOK_INT       "int"
-%token TOK_LONG      "long"
-%token TOK_REGISTER  "register"
-%token TOK_RESTRICT  "restrict"
-%token TOK_RETURN    "return"
-%token TOK_SHORT     "short"
-%token TOK_SIGNED    "signed"
-%token TOK_SIZEOF    "sizeof"
-%token TOK_STATIC    "static"
-%token TOK_STRUCT    "struct"
-%token TOK_SWITCH    "switch"
-%token TOK_TYPEDEF   "typedef"
-%token TOK_UNION     "union"
-%token TOK_UNSIGNED  "unsigned"
-%token TOK_VOID      "void"
-%token TOK_VOLATILE  "volatile"
-%token TOK_WCHAR_T   "wchar_t"
-%token TOK_WHILE     "while"
+%token ATOK_AUTO      "auto"
```

Comment:
Could you please elaborate as to why token names that ought to be local to a translation unit need to be renamed?

Reply:
This is a -Wodr

Reply:
Looks like `%define api.token.prefix {prefix}` would be the less-hacky solution, but that wouldn't work on macOS, which has bison < 3.0. So we will have to bite the bullet, thank you for coming up with this.

Reply:
> Looks like `%define api.token.prefix {prefix}` would be the less-hacky solution, but that wouldn't work on macOS, which has bison < 3.0. So we will have to bite the bullet, thank you for coming up with this.

The MacOS jobs in the CI pipeline are currently installing Bison version 3.8.2 from brew. There are commits following an `api.token.prefix` approach in this PR - https://github.com/diffblue/cbmc/pull/7878